### DISCUSSION TITLE
Test

### DISCUSSION THREAD

**Discussion by ProximalElk6186** [ProximalElk6186/xmb-linux#3] [Announcements]
Test discussion

**Comment by ProximalElk6186** [comment]
Test comment

**Reply by ProximalElk6186** [reply]
Test reply

**Reply by ProximalElk6186** [reply]
> Test reply

Test quote reply

**Reply by ProximalElk6186** [reply]
Test reply to thread

**Comment by 5d-code** [comment]
can you not make this a linux distro and just make it an electron app

**Reply by ProximalElk6186** [reply]
Really?

**Reply by 5d-code** [reply]
yes

**Reply by 5d-code** [reply]
or make it a de / wm IDK man I don't want to reinstall linux

**Reply by 5d-code** [reply]
I mean ur using html,css and Js sooooooooooo it can be an electron app

**Reply by ProximalElk6186** [reply]
And I don't need to fork a web browser just to do this project? Then this is a perfect solution. Gotta work on it next week.

**Reply by 5d-code** [reply]
well if you said
Recreate the SONY PLAYSTATION 3's XMB™ interface using HTML + CSS + JavaScript
then I instantly think Electron

**Reply by ProximalElk6186** [reply]
Well, I looked at electron's webpage and it looks like it's a perfect solution.

**Reply by 5d-code** [reply]
ik then why u denying that ur using electron and secondly you don't need to make a whole distro make it like steam where there is big picture mode. just make an electron app that is fullscreen and don't make a whole distro, its pointless

**Reply by ProximalElk6186** [reply]
About denying that I'm using Electron: I actually don't use it. I just got to know about Electron from you. About making a distro: yeah, it may be pretty pointless. Gotta update the readme file (tomorrow, not now, it's almost 10 p.m. in my country currently).
